subsys/usb/device_next/class/usbd_dfu_flash.c

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "usbd_msg.h"
+
+#include <zephyr/kernel.h>
+#include <zephyr/usb/usbd.h>
+#include <zephyr/usb/class/usbd_dfu.h>
+#include <zephyr/drivers/flash.h>
+#include <zephyr/storage/flash_map.h>
+#include <zephyr/dfu/flash_img.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(dfu_flash, CONFIG_USBD_DFU_LOG_LEVEL);
+
+/*
+ * This file implements the flash backend for the USB DFU implementation. The
+ * flash backend can serve up to two image slots, which are typically defined
+ * for the in-three boards in the Zephyr project.
```

Comment:
```suggestion
 * for the in-*tree* boards in the Zephyr project.
```

Reply:
Why?

Reply:
I removed the asteriks - I just wanted to mention the typo.